### DISCUSSION TITLE
Installazione app

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#567] [Richieste di aiuto] [answered]
Ciao come faccio ad installare l'app del registro del mio istituto
Cordiali saluti
Prof Coco Davide

**Comment by trinko** [comment, marked as answer]
L'app GiuaReg (solo versione Android) si può scaricare dalla pagina principale del progetto.
Questo è il link diretto al  file di installazione: [GiuaReg-3.0](https://github.com/iisgiua/giuaReg/raw/master/dist/giuaReg-v3.0.apk)

Una volta che la scarichi nel telefono, la mandi in esecuzione, vai nella configurazione e inserisci i dati necessari.
